README.md

```diff
@@ -18,8 +29,20 @@ Your database will be PSQL, and you will interact with it using [node-postgres](
 8. pg-format (dev-dependencies)
 9. husky (dev-dependencies)
 
-## Environmental Variables
+### Environmental Variables
 
 Please at the root directory of your local filesystem, create two new environment variable files `.env` for both development `.env.development` and test environments `.env.test`.
 
 Enter `PGDATABASE=nc_news_test` in `.env.test` directory while `PGDATABASE=nc_news` in `.env.development` directory.
+
+## Instruction on running test and seeding the local database
+
+1. Make sure your local machine reach the minimum requirement and all the npm packages are installed.
+2. Start up PostgreSQL in your local machine.
```

Comment:
They'll also need to type `npm install` at some point, or all this stuff below won't work